decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -16,13 +16,17 @@
   feature.settings(:global) do |settings|
     settings.attribute :vote_limit, type: :integer, default: 0
     settings.attribute :comments_always_enabled, type: :boolean, default: true
+    settings.attribute :proposal_answering_enabled, type: :boolean, default: true
+    settings.attribute :official_proposals_enabled, type: :boolean, default: true
+    settings.attribute :scoped_proposals_enabled, type: :boolean, default: true
```

Comment:
@oriolgual If the setting is not found it uses the default value so the migration is not needed.